src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlColumnEncryptionCspProvider.Windows.cs

```diff
@@ -419,12 +419,12 @@ private void GetCspProviderAndKeyName(string keyPath, bool isSystemOp, out strin
         /// Gets the provider type from a given CAPI provider name
         /// </summary>
         /// <param name="providerName">CAPI provider name</param>
-        /// <param name="keyPath">key path in the format of [CSP provider name]\[key name]</param>
+        /// <param name="keyPath">key path in the format of [CSP provider name]/[key name]</param>
         /// <param name="isSystemOp">Indicates if ADO.NET calls or the customer calls the API</param>
         /// <returns></returns>
         private int GetProviderType(string providerName, string keyPath, bool isSystemOp)
         {
-            string keyName = String.Format(@"SOFTWARE\Microsoft\Cryptography\Defaults\Provider\{0}", providerName);
+            string keyName = String.Format(@"SOFTWARE/Microsoft/Cryptography/Defaults/Provider/{0}", providerName);
```

Comment:
This may not be necessary.

Reply:
yes. you are right. I missed that part. Reverted.